.gitmodules

```diff
@@ -34,3 +34,8 @@
 	path = meta-intel
 	url = https://git.yoctoproject.org/git/meta-intel
 	branch = dunfell
+[submodule "meta-sca"]
```

Comment:
@toganlabs I noticed this submodule was not automatically cloned when I checked out this branch - I think it still needs the commit-id file of this submodule also checked in.